actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -21,12 +21,17 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.use_authenticated_cookie_encryption = false
     config.action_dispatch.perform_deep_munge = true
 
+    config.action_dispatch.legacy_encryptor_schemes = []
+    config.action_dispatch.legacy_verifier_schemes  = []
+
     config.action_dispatch.default_headers = {
       "X-Frame-Options" => "SAMEORIGIN",
       "X-XSS-Protection" => "1; mode=block",
       "X-Content-Type-Options" => "nosniff"
     }
 
+    config.action_dispatch.cookies_rotations = ActiveSupport::Messages::Configuration.new.cookies_rotations
```

Comment:
Looks like we're always calling this with `new.cookies_rotations`, then there's no point to the namespace handling in the `Configuration` class and we should just move up `Rotation` to `RotationConfiguration`.

Reply:
Should I convert this class to a Singleton? Or should I make `rotate` as well as the `signed` and `encrypted` arrays into class/module methods?

Reply:
Nope, let's keep it a regular class. Assigning it to `Rails.application.config` makes it a "singleton", i.e. accessible across the app.